ambry-api/src/main/java/com/github/ambry/config/CloudConfig.java

```diff
@@ -45,6 +50,7 @@
   public static final String VCR_PROXY_HOST = "vcr.proxy.host";
   public static final String VCR_PROXY_PORT = "vcr.proxy.port";
   public static final String VCR_CLUSTER_SPECTATOR_FACTORY_CLASS = "vcr.cluster.spectator.factory.class";
+  public static final String VCR_CROSS_COLO_REPLICATION_PEER_DATACENTERS = "vcr.cross.colo.replication.peer.datacenter";
```

Comment:
This is very long.  How about VCR_SOURCE_DATACENTERS and a comment explaining it?

Reply:
sure.